src/Platforms/CUDA_legacy/CMakeLists.txt

```diff
@@ -12,9 +12,9 @@
 set(CUDA_LEGACY_SRCS cuda_inverse.cu gpu_vector.cpp gpu_misc.cpp)
 
 if(NOT QMC_CUDA2HIP)
-  cuda_add_library(platform_cuda_legacy ${CUDA_LEGACY_SRCS})
-  cuda_add_cublas_to_target(platform_cuda_legacy)
-  target_link_libraries(platform_cuda_legacy PRIVATE ${CUDA_LIBRARIES})
+  add_library(platform_cuda_legacy ${CUDA_LEGACY_SRCS})
+  target_include_directories(platform_cuda_legacy PUBLIC ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
```

Comment:
`target_link_libraries(platform_cuda_runtime PUBLIC CUDA::cudart) `

Reply:
platform_cuda_runtime isn't part of this file. Did you mean ` target_link_libraries(platform_cuda_legacy PUBLIC CUDA::cublas)`?

Reply:
Yes. Was a copy paste mistake.

Reply:
changed.